### ISSUE TITLE
Put this repo inactive and link to the new biodiversity package

### ISSUE THREAD

**Issue by Jo-Schie** [mapme-initiative/mapme.forest#11]
Hi @goergen95 . I think we can alerady deactivate this repo and link to the new package. Or what do you think? Only thing here that is not there yet is the fragstats but I guess that would not justify an active maintenance, or?

**Comment by goergen95** [comment]
Agreed! mapme.biodiversity is indeed the better choice. Hopefully we can implement the fragmentation statistics via [landscapemetrics](https://r-spatialecology.github.io/landscapemetrics/index.html) later this year... I have no experiences with archiving repos. Will it still be visible afterwards? If yes, we should include a prominent note linking to mapme.biodiversity. If no, we could simply put it to the archive mode.

**Comment by Jo-Schie** [comment]
Not sure. No experience on my side either. I think it is still visible afterwards but read only and yes, in that case it makes sense to put a prominent note in the readme with the new Repo.
